omnigibson/scenes/interactive_traversable_scene.py

```diff
@@ -51,6 +52,7 @@ def __init__(
             not_load_object_categories (None or list): if specified, do not load these object categories into the scene
             load_room_types (None or list): only load objects in these room types into the scene
             load_room_instances (None or list): if specified, only load objects in these room instances into the scene
+            load_task_relevant_only (bool): Whether only task relevant objects (and building structure) should be loaded
```

Comment:
Sorry, you might have discussed this with Josiah offline. But I am not fully convinced that scene is somehow task-conditioned (it should be task agnostic imho). Could you explain your use case once again?

Reply:
It simplifies the task a lot by reducing the action space size for the object-centric primitives which makes it much easier for GPT etc. Use case is for quick evaluation of task policies without having to load everything. Also this way we can iterate on laptops etc. that can't normally load the whole scene.

The scene _file_ is definitely task conditioned (it has the task name in its name, you can't switch it to become the scene for another task, etc.) and there's no simple mechanism that will allow this kind of filtering to happen without having the scene/task objects talk to each other or the env do a bunch of plumbing, none of which beats this simple setup imo.